### PR TITLE
fix: Revised COVID-19 Google Mobility dataset

### PR DESCRIPTION
**Pull request by nlarge-google**
## Description

This DAG is failing in production and so this PR attempts to resolve the issues by both correcting the code and making the code consistent with existing implementations on this project.

Dataset: covid19_google_mobility
pipeline/s: mobility_report

## Checklist

Note: If an item applies to you, all of its sub-items must be fulfilled

- [x] **(Required)** This pull request is appropriately labeled
- [x] Please merge this pull request after it's approved
- [ ] I'm adding or editing a feature
- [x] I'm adding or editing a dataset
  - [x] The [Google Cloud Datasets team](mailto:cloud-datasets-onboarding@google.com) is aware of the proposed dataset
  - [x] I put all my code inside  `datasets/covid19_google_mobility>` and nothing outside of that directory
- [x] I'm refactoring or cleaning up some code

### CODE REVIEW

File: datasets/covid19_google_mobility/pipelines/mobility_report/pipeline.yaml [hunk 1]
```diff
@@ -35,100 +35,88 @@ dag:
...
-          SOURCE_FILE: "files/data.csv"
-          TARGET_FILE: "files/data_output.csv"
+          PIPELINE_NAME: "{{ var.json.covid19_google_mobility.mobility_report.pipeline_name }}"
+          SOURCE_URL: "{{ var.json.covid19_google_mobility.mobility_report.source_url }}"
+          SOURCE_FILE: "{{ var.json.covid19_google_mobility.mobility_report.source_file }}"
+          TARGET_FILE: "{{ var.json.covid19_google_mobility.mobility_report.target_file }}"
+          CHUNKSIZE: "{{ var.json.covid19_google_mobility.mobility_report.chunksize }}"
+          PROJECT_ID: "{{ var.value.gcp_project }}"
+          DATASET_ID: "{{ var.json.covid19_google_mobility.mobility_report.dataset_id }}"
+          TABLE_ID: "{{ var.json.covid19_google_mobility.mobility_report.table_id }}"
```
Comment (adlersantos) [inline datasets/covid19_google_mobility/pipelines/mobility_report/pipeline.yaml:53]: Let's parameterize only `PROJECT_ID`. Everything else we can make transparent to readers of our code (open source).

File: datasets/covid19_google_mobility/pipelines/mobility_report/pipeline.yaml [hunk 2]
```diff
@@ -35,100 +35,88 @@ dag:
...
+          TARGET_GCS_PATH: "{{ var.json.covid19_google_mobility.mobility_report.target_gcs_path }}"
+          SCHEMA_PATH: "{{ var.json.covid19_google_mobility.mobility_report.schema_path }}"
+          DROP_DEST_TABLE: "{{ var.json.covid19_google_mobility.mobility_report.drop_dest_table }}"
+          INPUT_FIELD_DELIMITER: "{{ var.json.covid19_google_mobility.mobility_report.input_field_delimiter }}"
+          REMOVE_SOURCE_FILE: "{{ var.json.covid19_google_mobility.mobility_report.remove_source_file }}"
+          DELETE_TARGET_FILE: "{{ var.json.covid19_google_mobility.mobility_report.delete_target_file }}"
```
Comment (adlersantos) [inline datasets/covid19_google_mobility/pipelines/mobility_report/pipeline.yaml:60]: We can expose these variables (except `TARGET_GCS_BUCKET`) so it's clear to readers of the code what values these variables should have

**Review by adlersantos — CHANGES_REQUESTED** [review]

File: datasets/covid19_google_mobility/pipelines/mobility_report/pipeline.yaml (hunk 2 above)
Comment (nlarge-google) [reply datasets/covid19_google_mobility/pipelines/mobility_report/pipeline.yaml]: Done.

File: datasets/covid19_google_mobility/pipelines/mobility_report/pipeline.yaml (hunk 1 above)
Comment (nlarge-google) [reply datasets/covid19_google_mobility/pipelines/mobility_report/pipeline.yaml]: Done

**Comment by nlarge-google** [comment]
@adlersantos Changes complete.  Please review.

**Review by adlersantos — APPROVED** [review]

**Pull request merged**